lib/gemstash/cli/info.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "gemstash"
+require "puma/control_cli"
+
+module Gemstash
+  class CLI
+    # This implements the command line setup task:
+    #  $ gemstash info
+    class Info < Gemstash::CLI::Base
+      include Gemstash::Env::Helper
```

Comment:
Why we need to include this helper here?

Reply:
I use it to access the current config return by gemstash_env.config, in list_config right below it.